lib/addresses.js

```diff
@@ -226,6 +226,38 @@ AddressController.prototype.transformAddressHistoryForMultiTxs = function(txinfo
   );
 };
 
+AddressController.prototype.paginatedUtxo = function(req, res, next) {
+  var self = this;
+
+  var options = {
+    from: parseInt(req.query.from) || parseInt(req.body.from) || 0
+  };
+
+  options.to = parseInt(req.query.to) || parseInt(req.body.to) || parseInt(options.from) + 1000;
+  if (options.from >= options.to) {
```

Comment:
why we can't getUtxo when 'from' equals 'to'?

Reply:
Good catch! That should really be allowed :). Will update